src/plugins/boolean/README.md

```diff
@@ -0,0 +1,70 @@
+- infos = Information about the boolean plugin is in keys below
+- infos/author = Name <name@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = recommended productive maintained reviewed conformant compatible coverage specific unittest shelltest tested nodep libc configurable preview nodoc concept
+- infos/metadata =
+- infos/description =
+
+## Example ##
+
+```
+% kdb umount /booltest
+% kdb mount bool.dump /booltest dump
+% kdb export /booltest ini -c meta=
+
+k1 = true
+k2 = tRUe
+k3 = i won't be true
+k4 = false
+k5 = FalsE
+k6 = i'm not false
+k7 = off
+k8 = on
+[k1]
+check/type = boolean
+[k2]
+check/type = boolean
+[k3]
+check/type = boolean
+[k4]
+check/type = boolean
+[k5]
+check/type = boolean
+[k6]
+check/type = boolean
+[k7]
+check/type = boolean
+[k8]
+check/type = boolean
+
+% kdb umount /booltest
+% kdb mount bool.dump /booltest dump boolean false='FALSE; OFF; 0',true='TRUE; ON; 1',set/true='this should be true',set/false='this should be false'
```

Comment:
default settings are 1 for true and 0 for false, and the accepted values are the same as in this example: "TRUE, ON, 1, FALSE, OFF, 0".